app/kumactl/cmd/install/install_control_plane.go

```diff
@@ -190,6 +143,76 @@ func newInstallControlPlaneCmd(pctx *kumactl_cmd.RootContext) *cobra.Command {
 	return cmd
 }
 
+func validateArgs(args InstallControlPlaneArgs) error {
+	if err := core.ValidateCpMode(args.KumaCpMode); err != nil {
+		return err
+	}
+	if args.KumaCpMode == core.Remote && args.Zone == "" {
+		return errors.Errorf("--zone is mandatory with `remote` mode")
+	}
+	if args.KumaCpMode == core.Remote && args.KdsGlobalAddress == "" {
+		return errors.Errorf("--kds-global-address is mandatory with `remote` mode")
+	}
+	if args.KdsGlobalAddress != "" && !strings.HasPrefix(args.KdsGlobalAddress, "grpcs://") {
```

Comment:
should this be a slightly more complex validation? Prefix is fine, but better check the full URL is valid (domain name or IPv4/v6, port, maybe even path?)